notebooks/community/managed_notebooks/Recommender Systems/Recommender-system_notebook.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    GCS_MODEL_PATH = "gs://your-bucket-name/recommender_systems/"
</pre><p>Must use the bucket created earlier in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>